website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,98 @@ The password auth method has the following additional attributes:
 
 - `min_password_length` - (required) The default is 8.
 
+### LDAP Auth Method Attributes
+
+The ldap auth method has the following additional attributes:
+
+- `state` - The state of the auth method; either inactive, active-private or
+  active-public
+
+- `start_tls` - (optional) If true, issues a StartTLS command after establishing
+  an unencrypted connection. Defaults to false.
+
+- `insecure_tls` - (optional) If true, skips LDAP server SSL certificate
+  validation - which is insecure and should be used with caution. Defaults to
+  false. 
+
+- `discover_dn` - (optional) If true, use anon bind to discover the bind DN
+  (Distinguished Name) of a user. Defaults to false.
+
+- `anon_group_search` - (optional) If true, use anon bind when performing LDAP
+  group searches. Defaults to false.
+
+- `upn_domain` - (optional) If set, the userPrincipalDomain used to construct
+  the UPN string for the authenticating user. The constructed UPN will appear as 
+  [username]@UPNDomain  Example: example.com, which will cause Boundary to
+  bind as username@example.com when authenticating the user.
+
+- `urls` - (required) The LDAP URLS that specify LDAP servers to connect to.
+  There must be at least one URL for each LDAP auth method. When attempting to
+  connect, the URLs are tried in the order specified. 
+
+- `user_dn` - (optional) If set, the base DN under which to perform user
+  search. Example: ou=Users,dc=example,dc=com
+
+- `user_attr` - (optional) If set, defines the attribute on a user's entry 
+  matching the login-name passed when the user authenticates.  Examples: cn, uid  
+
+- `user_filter` - (optional) If set, the go template used to construct a LDAP
+  user search filter. The template can access the following context variables:
+  [UserAttr, Username]. The default userfilter is ({{.UserAttr}}={{.Username}})
```

Comment:
The `_` separator in the option name implies to me that this is two words.
```suggestion
  [UserAttr, Username]. The default user filter is ({{.UserAttr}}={{.Username}})
```

Reply:
Added an underscore to be consistent with the name.